src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Base/App.default.cs

```diff
@@ -111,7 +178,27 @@ private static void RegisterRoutes(IViewRegistry views, IRouteRegistry routes)
 				{
 					new RouteMap("Main", View: views.FindByViewModel<MainModel>()),
 					new RouteMap("Second", View: views.FindByViewModel<SecondModel>()),
-				})
+				}
+			)
 		);
+#else
+		views.Register(
+			new ViewMap(ViewModel: typeof(ShellViewModel)),
+			new ViewMap<MainPage, MainViewModel>(),
+			new DataViewMap<SecondPage, SecondViewModel, Entity>()
+		);
+
+		routes.Register(
+			new RouteMap("", View: views.FindByViewModel<ShellViewModel>(),
+				Nested: new RouteMap[]
+				{
+					new RouteMap("Main", View: views.FindByViewModel<MainViewModel>()),
```

Comment:
If we change this to use FindByView instead of FindByViewModel then we don't need different code for reactive v mvvm

Reply:
these are "duplicated" because of the difference in name between Reactive Models and MVVM ViewModels